velox/common/base/tests/SimdUtilTest.cpp

```diff
@@ -491,4 +491,25 @@ TEST_F(SimdUtilTest, memcpyTime) {
   LOG(INFO) << "simd=" << simd << " sys=" << sys;
 }
 
+TEST_F(SimdUtilTest, testSimdStrStr) {
```

Comment:
Since this is an almost drop-in replacement for `strstr`, can you add another test to generate a few thousands of random string pairs with various length, and check our result is the same as `strstr`?  Also make sure you cover corner cases like larger needle than stack, etc.

Reply:
@Yuhta 
yeah,  I will add another pr later for simd_strstr later.